src/modules/markets/markets.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { MarketEntity } from './entities/market.entity';
+import { DbResponseHelper } from '../../helpers/db-response.helper';
+
+@Injectable()
+export class MarketsService {
+  constructor(
+    @InjectRepository(MarketEntity, "mssql-cis")
+    private readonly marketsRepository: Repository<MarketEntity>,
+  ) {}
+
+
+  async findAllUsingSP(active?: string): Promise<MarketEntity[]> {
+    let spResults = await this.marketsRepository.query('CIS_USP_READ_MARKETS')
+
+    if (active !== undefined) {
+      if (active === 'true') {
+        spResults = spResults.filter(results => results.ACTIVE_IND == 'Y')
```

Comment:
can we use tripple equal? i.e. `results.ACTIVE_IND === 'Y'` 
this will check both the type (string/number/etc) and the value

Reply:
done